docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+# Text
+
+There are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents` namespace contains the class.
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](../../get-started/nui/quickstart.md).
+
+**Figure: Text label positioned to top left**
+
+![Text label example, positioned to top left](media/TextLabelTopLeft.png)
+
+<a name="create"></a>
+### Create Text Label
+
+To create a text label:
+
+1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+
+    ```csharp
+    Window window = Window.Instance;
+    TextLabel label = new TextLabel("Hello World");
+    window.Add(label);
+    ```

```

Comment:
One of the reason that I asked to check https://github.com/Samsung/tizen-docs/pull/1009#discussion_r377386986 and https://github.com/Samsung/tizen-docs/pull/1009#discussion_r385701636is is below removed note.

\> [!CAUTION]
\> The `TextLabel` object can be only added to a `Window` object or a `View` object on a `Window` object.


This note is removed during the review. Is it really OK to go without it? Please check.
If the restriction that TextLabel can be added to Widonw or View is removed, we can skip this update.
If not, we should revert it back.

Reply:
@mijin85cho Since I am not the author, earlier there was note and same was removed by the author of this file. How am I expected to know what should be kept as author only removed it.

Reply:
Why I ask you to trace the updates by author is, it is removed according to your suggestion.

![image](https://user-images.githubusercontent.com/9958374/77627692-9b4e6e80-6f8a-11ea-859e-b433e746f09f.png)

Yes you are not the author and don't know whether the note can be removed or not.

And I think you often experienced that many authors mechanically apply the comments.
In this situation, it has possibility to make wrong updates when a reviewer's review is wrong.

For this note, you just asked your doubt,
and it just removed the content without further checking.

I'm worried about these kind of wrong editing directed by review comments,
so keep asking you restrain the sentence changing itself excepting grammar check.

I hope you keep in mind that your comment makes the modification,
so the updated result by your comment should be checked also.

Reply:
@mijin85cho  when the content was present I had a concern regarding its grammar and then, I thought the author might have changed mind to remove it, and with force pushes we tend to lose the original content so often. But going forward will enquire and question as to why it was removed.

Reply:
Also for me, I vaguely remember that I checked that the author himself has resolved the conversation, that made me think at that point of time that it may be no longer needed. I am sorry for presuming with common logic. 
<img width="713" alt="resolve" src="https://user-images.githubusercontent.com/43839737/77632393-f1fc7e00-6f73-11ea-98a2-f76a9cb1a8ef.PNG">

Reply:
I thought nawab-samsung's opinion was reasonable and removed it.
This note was only on the text label and I thought there would be no need to explain further about the Window and View. so remved it.
And, according to mijin85cho's opinion, I modified it like now. 
This is the same as the sample code for Create TextField and Create TextEditor.